orientation/faq.md

```diff
@@ -0,0 +1,215 @@
+
+The price of something is determined by an agreement of a buyer and seller. Absent sufficient buying and selling, price is unpredictable and volatile. Liquidity pools solve the price problem by underlying price discovery for an asset using a mathematical algorithm (bonding curve) combined with two pairs of tokens as liquidity for trading. In this way, liquidity pools allow for a kickstarting of liquidity and price discovery for cryptoassets without any intermediary.
+
+For Ocean, each published data asset with a datatoken has an associated liquidity pool. Participants can curate data sets by staking tokens in a liquidity pool, that is locking their Ocean with datatokens to create a token pool that people can trade against. In this way, the price of a datatoken can be determined by the market.
+Stake sufficient liquidity to allow for trading with minimal slippage - Ocean token is used to generate datatokens, with 10% of fees going to the publisher 10%. Usually people use FIAT to guide them on how much they want the asset to be priced initially. It is generally recommended to begin with a low price with high liquidity (datatoken) doing 5x or 10x more than the minimum recommended. This will improve liquidity and the price will typically go up over time.
+
+</details>
+
+<details>
+  <summary>What is staking</summary>
+
+Staking is the act of locking tokens to add liquidity to an Ocean datatoken liquidity pool. By staking, users are de facto curating datasets and becoming liquidity providers (LP) or “staker” for that particular datatoken liquidity pool.
+
+Stakers receive a portion of the transaction fees from trading activities that happen in the datatoken pool(s). By staking on datasets, stakers can speculate on attractive future returns on datasets that they deem as promising. Staking opportunities come with some inherent risks such as impermanent loss.
+
+</details>
+
+### Data Selling Questions
+
+<details>
+  <summary>How are organizations leveraging data sharing?</summary>
+
+For the most part organizations are leveraging data sharing to benefit from data monetization, however increasingly organizations are also sharing data in order to boost their progress on sustainability goals. For example, data aggregated from vehicles can not only bring new revenue streams to automotive firms but can also be used to battle pollution.
+
+</details>
+
+<details>
+  <summary>Does it pay to become a marketplace operator?</summary>
+
+Yes. Marketplace operators benefit from earning commission on marketplace transactions related to data consumption as well as staking. Ocean Market is primarily focussed on monetising data however it is also designed to handle the sale of any digital asset or service. As a result the total addressable market goes way beyond revenues from just selling data. Operating costs for an Ocean-powered marketplace are moderate and the base code is open source and available free of charge under the Apache 2 license.
```

Comment:
Hm, I think we should remove all references of staking, liquidity pools and dynamic pricing schema

Reply:
Yeah true, I've removed anything that mentions pools or dynamic pricing